crates/sui-core/src/transaction_input_checker.rs

```diff
@@ -155,11 +153,15 @@ where
         let gas_object = gas_object.ok_or(SuiError::ObjectNotFound {
             object_id: gas_payment_id,
         })?;
-        let gas_price = std::cmp::max(computation_gas_price, STORAGE_GAS_PRICE);
+        let storage_gas_price = store
```

Comment:
Let's add a TODO here to cache the storage_gas_price in a local variable, instead of reading it each time when checking a transaction.